### DISCUSSION TITLE
Feature test

### DISCUSSION THREAD

**Discussion by xicond** [paksuco/dusk-time-travel#1] [Q&A] [answered]
I'm writing API test, and test it use **Feature Test**, How to do time travel on it
let say API token must expire on 1 weeks

**Comment by shane-smith** [comment]
@xicond - did you solve your problem? If not, reply here with the contents of your test file and we can assist.

**Reply by shane-smith** [reply, marked as answer]
Due to the age of this thread I'll close it and you can always open a new issue if you're still having issues :)
